testing/str_test.py

```diff
@@ -12,7 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# pylint: disable=redefined-outer-name
+# pylint: disable=redefined-outer-name,W0703
+
+import sys
+
+
+def checkequal(expected, s, attr, *args):
```

Comment:
Sure, I will update. I copied those codes from CPython tests, I was lazy to rewrite it.